generated/json_data/qol.jsonc

```diff
@@ -9028,174 +9135,834 @@
                         }
                     ]
                 },
-                "Artifact Temple": {
-                    // Make Teleporter play sounds again when ridley dead
-                    "addConnections": [
-                        {
-                            "senderId": 1048578, // [SpawnPoint] Spawn point - NON-Teleporter Spawn
-                            "targetId": 2048598, // [Relay] Play Teleporter Sounds
-                            "state": "INACTIVE",
-                            "message": "ACTIVATE"
+                "Artifact Temple": {/* This, but it needs to be done in code.
```

Comment:
This is far from the correct place to request new features. Please either start a discussion in the discord server or [make an issue](https://github.com/toasterparty/randomprime/issues)